frontend/views/pages/Payments.vue

```diff
@@ -301,6 +314,12 @@ export default ({
         amount: this.withGroupCurrency(amount),
         count: this.paymentsTodo.length
       }
+    },
+    isCloseToDistributionTime () {
+      const dDay = this.periodAfterPeriod(this.groupSettings.distributionDate)
+      const warningDate = addTimeToDate(dDay, -7 * DAYS_MILLIS)
+
+      return Date.now() >= new Date(warningDate).getTime() && Date.now() < dDay.getTime()
```

Comment:
`dDay.getTime()` will throw an exception here because `periodAfterPeriod` returns a string, not a date, and needs to be converted using the `dateFromPeriodStamp` function from `time.js`.

So to fix line 319 should look like this:

```js
const dDay = dateFromPeriodStamp(this.periodAfterPeriod(this.groupSettings.distributionDate))
```

Sorry for not catching this in my previous review. You can use the Date & Time System Preferences to test this stuff locally.